app/app.jsx

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
+import { connect } from 'react-redux'
 import './scss/style.scss';
 import 'normalize.css';
 
-class App extends React.Component {
-   render() {
-      return (
-         <div>
-            Hello World!!!
-         </div>
-      );
-   }
-}
+/* import components */
+import Main from './containers/main.jsx';
+import Nav from './containers/nav.jsx';
+
+let App = () => {
+
+  return (
+    <div className="page">
+      <Nav />
+      <Main />
+    </div>
+  );
+};
+
+App = connect()(App)
```

Comment:
ok i've deleted that, but not sure that i get why the "main" container needs to have it but the app doesn't? Is it because main is a level deeper and therefore handles the state changes?

not sure if what i said makes any sense...